vuepress/docs/next/docs/compose/ecm-overview.md

```diff
@@ -0,0 +1,20 @@
+# Entando Component Manager
+
+​​An Entando Application is composed of the Entando App Builder, Entando App Engine, and Entando Component Manager. The Entando Component Manager (ECM) is a service that links the [Entando Component Repository](ecr-overview.md) (ECR) of the App Builder to the core application instance. It appears as `quickstart-cm-deployment` in the Kubernetes pod list:
+
+![pods.png](./img/pods.png)
+
+The ECM communicates with the Entando Kubernetes service to populate the ECR with the bundles deployed as [Custom Resources](../consume/custom-resources.md) in the Entando cluster. These bundles are available to the application and can be managed from within the ECR. 
```

Comment:
"deployed as... in the Entando cluster" change to "available as... in the Entando namespace. These bundles can be installed in the application and managed from within the ECR."

Not sure why I think that reads better but part of it is you don't deploy a Custom Resource - even though that's exactly the term we use in the UI for a bundle coming from the hub. It just feels wrong when it's said directly about a CR.